platforms/corda-ent/charts/signer-setup/templates/job.yaml

```diff
@@ -0,0 +1,215 @@
+            cp CM-FILES/signer.conf etc/signer.conf;
+            mkdir DATA/key-stores DATA/trust-stores DATA/crl-files
+            time java -Xmx{{ .Values.cordaJarMx }}G -jar {{ .Values.jarPath }}/pkitool.jar --config-file {{ .Values.configPath }}/pki.conf
+
+            #creating a dummy file to perform check if last line is executed or not.
+            touch DATA/done.txt
+          volumeMounts:
+          - name: pkitool-certs-keys
+            mountPath: /opt/corda/DATA
+          - name: pkitool-signer-etc
+            mountPath: /opt/corda/etc
+          - name: pkitool-conf
+            mountPath: /opt/corda/CM-FILES/pki.conf
+            subPath: pki.conf
+          - name: signer-conf
+            mountPath: /opt/corda/CM-FILES/signer.conf
+            subPath: signer.conf
+          resources:
+            requests:
+              memory: {{ .Values.cordaJarMx }}G
+            limits:
+              memory: {{ add .Values.cordaJarMx 2 }}G
+        - name: store-certs-in-vault
+          image: {{ .Values.image.initContainerName }}
+          imagePullPolicy: Always
+          env:
+             - name: VAULT_ADDR
+               value: {{ $.Values.vault.address }}
+             - name: KUBERNETES_AUTH_PATH
+               value: {{ $.Values.vault.authpath }}
+             - name: VAULT_APP_ROLE
+               value: {{ $.Values.vault.role }}
+             - name: BASE_DIR
+               value: {{ $.Values.volume.baseDir }}
+             - name: RT_SECRET_PREFIX
+               value: {{ .Values.vault.tokensecretprefix }}
+             - name: CERTS_SECRET_PREFIX
+               value: {{ .Values.vault.certsecretprefix }}
+             - name: JAVA_OPTIONS
+               value: -Xmx512m
+          command: ["sh", "-c"]
+          args:
+          - |-
+            #!/usr/bin/env sh
+
+            OUTPUT_PATH=${BASE_DIR}
+
+            # setting up env to get secrets from vault
+            KUBE_SA_TOKEN=$(cat /var/run/secrets/kubernetes.io/serviceaccount/token)
+            VAULT_TOKEN=$(curl -sS --request POST ${VAULT_ADDR}/v1/auth/${KUBERNETES_AUTH_PATH}/login -H "Content-Type: application/json" -d '{"role":"vault-role","jwt":"'"${KUBE_SA_TOKEN}"'"}' | jq -r 'if .errors then . else .auth.client_token end')
+
+            # get the root token from the vault, to upload secrets on vault
+            LOOKUP_PWD_RESPONSE=$(curl -sS --header "X-Vault-Token: ${VAULT_TOKEN}" ${VAULT_ADDR}/v1/${RT_SECRET_PREFIX} | jq -r 'if .errors then . else . end')
+            ROOT_TOKEN=$(echo ${LOOKUP_PWD_RESPONSE} | jq -r '.data["rootToken"]')
+
+            # perform check if crl certificates are ready or not, and upload certificate into vault when ready
+            COUNTER=1
+            cd ${BASE_DIR}/DATA/crl-files
+            while [ "$COUNTER" -lt {{ $.Values.healthcheck.readinessthreshold }} ]
+            do
+                if [ -e tls.crl ] && [ -e root.crl ] && [ -e ssl.crl ] && [ -e ../done.txt ]
+                then
```

Comment:
Check https://blockchain-automation-framework.readthedocs.io/en/develop/architectureref/certificates_path_list_corda_ent.html for Vault paths

Reply:
Issue #785 created for this